src/DevBetterWeb.Web/DevBetterWeb.Web.csproj

```diff
@@ -1,68 +1,71 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-  <PropertyGroup>
```

Comment:
See if you can reformat this to match what was there before so every line isn't different due to whitespace. There should be a .editorconfig file which sets spaces per tab to 2. You seem to have use 4 (or more).

Reply:
this is another thing I didn't change. Seems to me it is properly nested. (PropertyGroup is inside Project and thus _should_ be indented farther, and that holds true for the whole file)